### PR TITLE
FLTK v1.3.5

### PR DESCRIPTION
**Pull request by McNeight**
Update FLTK to version 1.3.5
Explicitly define default configuration settings

### CODE REVIEW

File: ports/fltk/portfile.cmake [hunk 1]
```diff
@@ -1,14 +1,14 @@
 include(vcpkg_common_functions)
-set(SOURCE_PATH ${CURRENT_BUILDTREES_DIR}/src/fltk-1.3.4-1)
+set(SOURCE_PATH ${CURRENT_BUILDTREES_DIR}/src/fltk-1.3.5)
 vcpkg_download_distfile(ARCHIVE
-    URLS "http://fltk.org/pub/fltk/1.3.4/fltk-1.3.4-1-source.tar.gz"
+    URLS "https://fltk.org/pub/fltk/1.3.5/fltk-1.3.5-source.tar.gz"
     FILENAME "fltk.tar.gz"
```
Comment (cenit) [inline ports/fltk/portfile.cmake]: Use a local filename that contains the version. Otherwise during upgrade vcpkg will complain about SHA mismatch

Comment (vicroms) [inline ports/fltk/portfile.cmake]:
```suggestion
    FILENAME "fltk-1.3.5.tar.gz"
```

**Review by vicroms — CHANGES_REQUESTED** [review]

File: ports/fltk/portfile.cmake (hunk 1 above)
Comment (McNeight) [reply ports/fltk/portfile.cmake]: Applied patch from @vicroms

**Comment by dan-shaw** [comment]
Thanks for the PR!

**Pull request merged**